test/query.js

```diff
@@ -123,7 +123,7 @@ test('seed', (t) => {
 })
 
 test('announce', (t) => {
-  messages.getContentAnnounce(metafeedKeys.id, (err, content) => {
+  messages.getContentAnnounce(metafeedKeys, (err, content) => {
```

Comment:
There must have been a bug here before?

Reply:
Yes there was. The checking is done correctly in `test/messages.js` but in this file we want to test the query APIs so we just used `getContentAnnounce` to create the inputs, but we didn't validate the input. Anyway whatever :)